java/src/jmri/jmrit/display/layoutEditor/LayoutBlock.java

```diff
@@ -229,7 +229,7 @@ public void setBlockExtraColor(Color color) {
         JmriColorChooser.addRecentColor(color);
     }
 
-    // TODO: @Deprecated // Java standard pattern for boolean getters is "UseExtraColor()"
+    // TODO: Java standard pattern for boolean getters is "UseExtraColor()"
```

Comment:
It might be good to change "UseExtraColor()" to "useExtraColor()" (first lower case character) so that the developer that finally fixes this doesn't give the method an upper case name. But if you don't want to, it's fine to leave it as is.